relayer/src/relay_merkle_roots.rs

```diff
@@ -235,6 +169,83 @@ impl MerkleRootRelayer {
     }
 }
 
+pub(crate) async fn sync_authority_set_id(
```

Comment:
Maybe move it into some `common` module?

Reply:
Extracted